File: simple_vel_ctrl/src/vel_ctrl_node.cpp
#include <ros/ros.h>
#include <geometry_msgs/Twist.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "vel_ctrl_node");

    ros::NodeHandle nodeHandle;

    ros::Publisher publisher = nodeHandle.advertise<geometry_msgs::Twist>("/turtle1/cmd_vel", 3);

    ros::Rate rate = ros::Rate(10);

    while(ros::ok())
    {
        geometry_msgs::Twist vel_msg;
        vel_msg.linear.x = 0.5;
        vel_msg.angular.z = 0.1;
        publisher.publish(vel_msg);

        rate.sleep();

    }
    return 0;
}

File: simple_vel_ctrl/scripts/vel_ctrl_node.py
#!/usr/bin/env python3

import rospy
from geometry_msgs.msg import Twist

if __name__ == '__main__':
    rospy.init_node("vel_ctrl_node")
    
    publisher = rospy.Publisher("turtle1/cmd_vel", Twist, queue_size=3)
    
    rate = rospy.Rate(10)
    
    while not rospy.is_shutdown():
        vel_msg = Twist()
        vel_msg.linear.x = 0.5
        vel_msg.angular.z = 0.1
        publisher.publish(vel_msg)
        
        rate.sleep()